<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>623</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Finance tracker</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:0, y2:0, stop:0 rgba(0, 102, 102, 255), stop:1 rgba(0, 191, 255, 255));
font-family: Roboto;</string>
  </property>
  <widget class="QWidget" name="main_widget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="header_container">
      <item>
       <widget class="QFrame" name="balance_frame">
        <property name="styleSheet">
         <string notr="true">background-color: rgba(255, 255, 255, 30);
border: 1px solid rgba(255, 255, 255, 40);
border-radius: 6px;</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>8</number>
         </property>
         <property name="topMargin">
          <number>8</number>
         </property>
         <property name="rightMargin">
          <number>8</number>
         </property>
         <property name="bottomMargin">
          <number>8</number>
         </property>
         <item>
          <widget class="QLabel" name="balance_title_lbl">
           <property name="styleSheet">
            <string notr="true">color: #c8fafa;
font-weight: bold;
font-size: 18px;
background-color: none;
border: none;
</string>
           </property>
           <property name="text">
            <string>Текущий баланс</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="balance_lbl">
           <property name="styleSheet">
            <string notr="true">color: #c8fafa;
font-size: 28px;
background-color: none;
border: none;
</string>
           </property>
           <property name="text">
            <string>1000</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="income_container">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>7</number>
           </property>
           <item>
            <widget class="QLabel" name="income_icon">
             <property name="maximumSize">
              <size>
               <width>24</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: none;
border: none;
padding-top: 10px;</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="pixmap">
              <pixmap resource="icon/res-rc.qrc">:/icons/arrow_insert.svg</pixmap>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="income_lbl">
             <property name="styleSheet">
              <string notr="true">color: #c8fafa;
font-size: 16px;
background-color: none;
border: none;
font-weight: bold;
padding-top: 10px;</string>
             </property>
             <property name="text">
              <string>Доход</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="income_balance_lbl">
           <property name="styleSheet">
            <string notr="true">color: #c8fafa;
font-size: 22px;
background-color: none;
border: none;
</string>
           </property>
           <property name="text">
            <string>1000</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="outcome_container">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>7</number>
           </property>
           <item>
            <widget class="QLabel" name="outcome_icon">
             <property name="maximumSize">
              <size>
               <width>24</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: none;
border: none;</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="pixmap">
              <pixmap resource="icon/res-rc.qrc">:/icons/call_received.svg</pixmap>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="outcome_lbl">
             <property name="styleSheet">
              <string notr="true">color: #c8fafa;
font-size: 16px;
background-color: none;
border: none;
font-weight: bold;</string>
             </property>
             <property name="text">
              <string>Затраты</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="outcome_balance_lbl">
           <property name="styleSheet">
            <string notr="true">color: #c8fafa;
font-size: 22px;
background-color: none;
border: none;
</string>
           </property>
           <property name="text">
            <string>1000</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="category_frame">
        <property name="styleSheet">
         <string notr="true">background-color: rgba(255, 255, 255, 30);
border: 1px solid rgba(255, 255, 255, 40);
border-radius: 6px;</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>8</number>
         </property>
         <property name="topMargin">
          <number>8</number>
         </property>
         <property name="rightMargin">
          <number>8</number>
         </property>
         <property name="bottomMargin">
          <number>8</number>
         </property>
         <item>
          <widget class="QLabel" name="category_title_lbl">
           <property name="styleSheet">
            <string notr="true">color: #c8fafa;
font-weight: bold;
font-size: 18px;
background-color: none;
border: none;
</string>
           </property>
           <property name="text">
            <string>Категории</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="groceries_container">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="groceries_icon">
             <property name="maximumSize">
              <size>
               <width>24</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: none;
border: none;
</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="pixmap">
              <pixmap resource="icon/res-rc.qrc">:/icons/shopping_cart.svg</pixmap>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="groceries_lbl">
             <property name="styleSheet">
              <string notr="true">color: #c8fafa;
font-weight: bold;
font-size: 14px;
background-color: none;
border: none;
</string>
             </property>
             <property name="text">
              <string>Продукты</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="groceries_balance">
             <property name="styleSheet">
              <string notr="true">color: #c8fafa;
font-size: 16px;
background-color: none;
border: none;
</string>
             </property>
             <property name="text">
              <string>1000</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="marketplace_container">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="marketplace_icon">
             <property name="maximumSize">
              <size>
               <width>24</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: none;
border: none;
</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="pixmap">
              <pixmap resource="icon/res-rc.qrc">:/icons/cloud.svg</pixmap>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="marketplace_lbl">
             <property name="styleSheet">
              <string notr="true">color: #c8fafa;
font-weight: bold;
font-size: 14px;
background-color: none;
border: none;
</string>
             </property>
             <property name="text">
              <string>Маркетплейсы</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="marketplace_balance">
             <property name="styleSheet">
              <string notr="true">color: #c8fafa;
font-size: 16px;
background-color: none;
border: none;
</string>
             </property>
             <property name="text">
              <string>1000</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="transport_container">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="transport_icon">
             <property name="maximumSize">
              <size>
               <width>24</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: none;
border: none;
</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="pixmap">
              <pixmap resource="icon/res-rc.qrc">:/icons/bus.svg</pixmap>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="transport_lbl">
             <property name="styleSheet">
              <string notr="true">color: #c8fafa;
font-weight: bold;
font-size: 14px;
background-color: none;
border: none;
</string>
             </property>
             <property name="text">
              <string>Транспорт</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="transport_balance">
             <property name="styleSheet">
              <string notr="true">color: #c8fafa;
font-size: 16px;
background-color: none;
border: none;
</string>
             </property>
             <property name="text">
              <string>1000</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="entertainment_container">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="entertainment_icon">
             <property name="maximumSize">
              <size>
               <width>24</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: none;
border: none;
</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="pixmap">
              <pixmap resource="icon/res-rc.qrc">:/icons/sports.svg</pixmap>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="entertainment_lbl">
             <property name="styleSheet">
              <string notr="true">color: #c8fafa;
font-weight: bold;
font-size: 14px;
background-color: none;
border: none;
</string>
             </property>
             <property name="text">
              <string>Развлечения</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="entertainment_balance">
             <property name="styleSheet">
              <string notr="true">color: #c8fafa;
font-size: 16px;
background-color: none;
border: none;
</string>
             </property>
             <property name="text">
              <string>1000</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="other_container">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="other_icon">
             <property name="maximumSize">
              <size>
               <width>24</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: none;
border: none;
</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="pixmap">
              <pixmap resource="icon/res-rc.qrc">:/icons/more.svg</pixmap>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="other_lbl">
             <property name="styleSheet">
              <string notr="true">color: #c8fafa;
font-weight: bold;
font-size: 14px;
background-color: none;
border: none;
</string>
             </property>
             <property name="text">
              <string>Другое</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="other_balance">
             <property name="styleSheet">
              <string notr="true">color: #c8fafa;
font-size: 16px;
background-color: none;
border: none;
</string>
             </property>
             <property name="text">
              <string>1000</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="buttons_container">
      <item>
       <widget class="QPushButton" name="new_btn">
        <property name="styleSheet">
         <string notr="true">QPushButton {
color: #c8fafa;
background-color: rgba(255, 255, 255, 30);
border: 1px solid rgba(255, 255, 255, 40);
border-radius: 6px;
width: 230px;
height: 50px;
}
QPushButton:hover {
background-color: rgba(255, 255, 255, 40);
}
QPushButton:pressed {
background-color: rgba(255, 255, 255, 70);
}

</string>
        </property>
        <property name="text">
         <string>Новая операция</string>
        </property>
        <property name="icon">
         <iconset resource="icon/res-rc.qrc">
          <normaloff>:/icons/add.svg</normaloff>:/icons/add.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="edit_btn">
        <property name="styleSheet">
         <string notr="true">QPushButton {
color: #c8fafa;
background-color: rgba(255, 255, 255, 30);
border: 1px solid rgba(255, 255, 255, 40);
border-radius: 6px;
width: 230px;
height: 50px;
}
QPushButton:hover {
background-color: rgba(255, 255, 255, 40);
}
QPushButton:pressed {
background-color: rgba(255, 255, 255, 70);
}

</string>
        </property>
        <property name="text">
         <string>Редактировать операцию</string>
        </property>
        <property name="icon">
         <iconset resource="icon/res-rc.qrc">
          <normaloff>:/icons/edit.svg</normaloff>:/icons/edit.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="delete_btn">
        <property name="styleSheet">
         <string notr="true">QPushButton {
color: #c8fafa;
background-color: rgba(255, 255, 255, 30);
border: 1px solid rgba(255, 255, 255, 40);
border-radius: 6px;
width: 230px;
height: 50px;
}
QPushButton:hover {
background-color: rgba(255, 255, 255, 40);
}
QPushButton:pressed {
background-color: rgba(255, 255, 255, 70);
}

</string>
        </property>
        <property name="text">
         <string>Удалить операцию</string>
        </property>
        <property name="icon">
         <iconset resource="icon/res-rc.qrc">
          <normaloff>:/icons/delete.svg</normaloff>:/icons/delete.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableWidget" name="table_container">
      <property name="styleSheet">
       <string notr="true">QTableView {
background-color: rgba(255, 255, 255, 30);
border: 1px solid  rgba(255, 255, 255, 40);
border-bottom-left-radius: 6px;
border-bottom-right-radius: 6px;
}
QTableView::section {
background-color: rgb(53, 53, 53);
color: #c8fafa;
border: none;
height: 50px;
font-size: 14px;
}

QTableView::item {
border-style: none;
border-bottom:  rgba(255, 255, 255, 50);
}

QTableView::item:selected {
border: none;
color: #E0F7FA;
background-color: rgba(255, 255, 255, 50);
}</string>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
      </property>
      <property name="showGrid">
       <bool>false</bool>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>135</number>
      </attribute>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="icon/res-rc.qrc"/>
 </resources>
 <connections/>
</ui>
